gui/fly-to-locations-tab.html

```diff
@@ -1,5 +1,52 @@
-<div id="celestial-bodies" class="row">
+<nav id="navi" class="hidden">
+    <div class="row nav nav-tabs" id="navigation-tabs" role="tablist">
+        <a class="nav-item nav-link active col-6"
+            id="nav-celestial-bodies"
+            data-toggle="tab"
+            href="#tab-celestial-bodies"
+            role="tab"
+            aria-controls="tab-celestial-bodies"
+            aria-selected="true">
+                Celestial Bodies
+        </a>
+        <a class="nav-item nav-link col-4"
+            id="nav-minimap"
+            data-toggle="tab"
+            href="#tab-minimap"
+            role="tab"
+            aria-controls="tab-minimap"
+            aria-selected="false">
+                Minimap
+        </a>
+    </div>
+</nav>
+<div class="tab-content">
+    <div id="tab-celestial-bodies" 
+         class="tab-pane fade show active"
+         role="tabpanel"
+         aria-labelledby="nav-celestial-bodies">
+        <div id="celestial-bodies" class="row"></div>
+    </div>
+    <div id="tab-minimap" 
+         class="tab-pane fade"
+         role="tabpanel"
+         aria-labelledby="nav-minimap">
+        <div id="minimap" class="row" onmouseover="window.dispatchEvent(new Event('resize'))">minimap</div>
```

Comment:
This displays a text over the minimap and can be removed.
```suggestion
        <div id="minimap" class="row" onmouseover="window.dispatchEvent(new Event('resize'))"></div>
```